docs/tutorials/run_emod.md

```diff
@@ -0,0 +1,280 @@
+
+    ```
+    idmtools container status 8ae683e4-8af6-4cd9-b8a3-5c63a5a10a17
+    ```
+
+says get status on experiment "8ae683e4-8af6-4cd9-b8a3-5c63a5a10a17".
+
+If you execute that line, you should see something similar to:
+
+```
+INI File Found: C:\work\emodpy-training\idmtools.ini
+```
+
+```
+Experiment Directory:
+c:/work/emodpy-training/emodpy-jobs/my_first_suite_2cf29edc-6b4d-459b-9542-3eae5c59a151/my_first_suite_8ae683e4-8af6-4cd
+9-b8a3-5c63a5a10a17
+```
+
+```
+Simulation Count: 1
+```
+
+```
+SUCCEEDED (1)
+FAILED (0)
+RUNNING (0)
+PENDING (0)
+```
+
+__Please note that this only works when using the Container Platform.  To see status
+when using other platforms, please see the
+[Command Line Interface documentation](https://docs.idmod.org/projects/idmtools/en/latest/cli/cli_index.html)
+ for your platform.__
+
+## The files produced when running EMOD
+
+When using the Container and SLURM platforms, you can access the directories where the
+experiments and simulations are running.  In our case, we are using the Container Platform
+and our idmtools.ini file told emodpy-workflow to put the files into a local folder
+called "emodpy-jobs".  If we navigate into this folder, we can see all of the files
+associated with running the experiment and simulations.  Let's go investigate.
+
+1. From the main project directory execute the following commands:
+
+    ```
+    cd emodpy-jobs
+    ls
+    ```
+    Note: Use "dir" on Windows instead of "ls"
+
+    You should see something similar to:
+
+    ```
+    Volume in drive C is Windows
+    Volume Serial Number is D277-F0D1
+    ```
+
+    ```
+    Directory of C:\work\emodpy-training\emodpy-jobs
+    ```
+
+    ```
+    07/22/2025  11:36 AM    <DIR>          .
+    07/22/2025  11:36 AM    <DIR>          ..
+    07/22/2025  11:36 AM    <DIR>          my_first_suite_2cf29edc-6b4d-459b-9542-3eae5c59a151
+                0 File(s)              0 bytes
+                3 Dir(s)  405,100,503,040 bytes free
+    ```
+
+    Notice the directory `my_first_suite_2cf29edc-6b4d-459b-9542-3eae5c59a151` starts
+    with the name we gave the suite.
+
+2. Look in the this directory by executing commands similar to the
+following - your directory name is likely different:
+
+    ```
+    cd my_first_suite_2cf29edc-6b4d-459b-9542-3eae5c59a151
+    ls
+    ```
+    Note: Use "dir" on Windows instead of "ls"
+
+    You should see something similar to the following:
+
+    ```
+    07/22/2025  11:36 AM    <DIR>          .
+    07/22/2025  11:36 AM    <DIR>          ..
+    07/22/2025  11:36 AM               476 metadata.json
+    07/22/2025  11:36 AM    <DIR>          my_first_suite_8ae683e4-8af6-4cd9-b8a3-5c63a5a10a17
+                1 File(s)            476 bytes
+                3 Dir(s)  403,785,535,488 bytes free
+    ```
+
+    Yes, it looks very similar, but this is the experiment directory.
+
+3. 
+
+
+
+
+d.	Have the user inspect the suite, experiment, and simulation folders
```

Comment:
Perspective shift--refer to the user in second-person perspective "you do X, Y, Z."